static/app/views/settings/components/dataScrubbing/utils.tsx

```diff
@@ -120,6 +124,11 @@ const valueSuggestions: Array<SourceSuggestion> = [
     value: '$user.ip_address',
     description: t('the user IP address'),
   },
+  {
+    type: SourceSuggestionType.VALUE,
+    value: '$span.description',
+    description: t('human readable description of a span'),
```

Comment:
I think this aligns better with the style of descriptions:
```suggestion
    description: t('the span description'),
```
"human readable" may be confusing for a field

Reply:
Agreed, I copied this from the Relay documentation on the field :D